engine/api/database/cli.go

```diff
@@ -23,8 +31,17 @@ var DBCmd = &cobra.Command{
 var upgradeCmd = &cobra.Command{
 	Use:   "upgrade",
 	Short: "Upgrade schema",
-	Long:  "Migrates the database to the most recent version available.",
-	Run:   upgradeCmdFunc,
+	Long: `Migrates the database to the most recent version available.
+
+	Example of usage:
```

Comment:
do you know that cobra and our wrapper support "Example": 
https://godoc.org/github.com/spf13/cobra#Command

https://github.com/ovh/cds/blob/master/cli/types.go#L102